test/data/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from box import Box
+from pathlib import Path
+import numpy as np
+
+
+@pytest.fixture
+def config():
+    cfg = Box.from_yaml(filename=Path("test/configs/test_config.yaml"))
+    cfg.dataset.type = 'train'
+    return cfg
+
+
+@pytest.fixture
+def dummy_input():
```

Comment:
Should be single image because our transformer would be called in `Dataset.__getitem__`.

Reply:
Yes. But the dummy_input would be used in testing dataloader as well, which need to yield a batch of data.

Reply:
Maybe add a new argument `size` to specify the dummy_input size for dataloader and transformers?

Reply:
Updated.